### ISSUE TITLE
Feature Request: Use Phone View when browser width is below 768px i.e. Phone size

### ISSUE THREAD

**Issue by JonJam** [Microsoft/PowerBI-JavaScript#147]
When a report is embedded into a responsive website, it would be ideal if the report automatically switched to [Phone View](https://powerbi.microsoft.com/en-us/documentation/powerbi-service-create-dashboard-phone-view/) when the browser window width was below a minimum value e.g. 768 px (Bootstrap's extra small device width).

Rather than currently where the report is scaled to fit the small browser width, rendering it unreadable.

**Comment by jmacweb** [comment]
Agreed, we are having to create multiple reports and use javascript to display the correct version, which is not ideal for the User experience.

**Comment by ali-hamud** [comment]
Hi,

An embedded report is responsive if its container is.
That means, you can define a responsive div using a css @media, and pass its id to power.embed then, try to resize the page and you will see the report size changes.

For example please see our live sample. It's responsive and you can see it in phone, tables and any screen you want

**Comment by JonJam** [comment]
@ali-hamud thanks for the pointers.

I probably didn't explain myself very well; if I use the live sample and make my browser window the same width as an iPhone 6 - 375px.

The report is responsive, however it looks as follows:

![image](https://cloud.githubusercontent.com/assets/1730684/23060334/32116750-f4f5-11e6-9627-c0d11c3f47bc.png)

It may be just me but I struggle to read any of the detail in the report, hence it would be better if the report switched to the Mobile View as the details could be more easily read.

**Comment by cruise-insights** [comment]
@ali-hamud  @JonJam @jmacweb 

I hope the wish is  : https://github.com/Microsoft/PowerBI-JavaScript/issues/137

**Comment by jmacweb** [comment]
I don’t we are being clear here…Yes the report will resize by shrinking the same proportions which becomes unreadable because too much info is in the report or screen.  Hence, the reason you guys created the mobile version, this version should be displayed instead when the screen is shrunk.

Also, if you could auto adjust the layout of the report elements that would work.  Just like a bootstrap grid.

Other helpful UI elements would be if we could send a report to a dashboard and use these in our pages, but only if you can have the Filters apply to all the dashboards.  Currently, we only see how to update reports based on filters in that specific report.  I would love to create individual reports on a dashboard with a dashboard level filter.  Based on your mobile experience this would solve the layout issues with very busy reports.

**Comment by amigup** [comment]
@noanu - Hello Noa, Is there any timeframe when this feature would be released?

**Comment by robrien6** [comment]
@noanu  - this would be awesome when delivered....

**Comment by robrien6** [comment]
@noanu hi, any update on this facility??

**Comment by epopov** [comment]
I vote for this feature too

**Comment by KimBurgess** [comment]
For anyone else awaiting support, hopefully this may be of use: https://github.com/acaprojects/powerbi-responsive.

Still plenty of improvements to be made, but base implementation should do what it says on the tin. Would love PR's and issues for anything funky you spot. I'm looking forward to this being made redundant by official support for responsive report layouts, but until then it may help to tied you over.

**Comment by kxon** [comment]
In 'acaprojects/powerbi-responsive' we have to duplicate pages in one report. In the future it can be problem because we have to make change in every page. Am I right?

Did something change in official 'Microsoft/PowerBI-JavaScript' ? can we call mobile view of power bi report?

**Comment by ali-hamud** [comment]
Hi,
Mobile View is not supported yet in embedded.
We are working on something which we will release on December which allows you to define the report layout on runtime.
This will give you an option to write fully responsive app, no matter how the report is saved.

Mobile layout is in our backlog as well, but I have no ETA for this.

**Comment by kxon** [comment]
@ali-hamud thanks for your answer. I'm looking forward
